### PR TITLE
feat: command menu v2 ui components

### PR DESCRIPTION
**Pull request by charislam**
Add all the UI components needed for Command Menu v2, and document it on the design system site.

### CODE REVIEW

**Comment by MildTomato** [comment]
To maybe improve this, maybe support for a Trigger component, so that all the right aria labels are used. See the [Dialog component](http://localhost:3003/design-system/docs/components/dialog)

Screenshot 2024-07-11 at 3 44 14 PM

**Review by MildTomato — APPROVED** [review]
Approved.

No changes to apps directly from what I can see.

**Comment by fsansalvadore** [comment]
The cmdk looked a bit funky and docs had old nav.
I resolved a small conflict and now it's up to date.

**Review by fsansalvadore — APPROVED** [review]

File: packages/common/hooks/index.ts [hunk 1]
```diff
@@ -8,3 +8,4 @@ export * from './useParams'
 export * from './useSearchParamsShallow'
 export * from './useTelemetryProps'
 export * from './useThemeSandbox'
+export * from './useDragToClose'
```
Comment (fsansalvadore) [inline packages/common/hooks/index.ts:11]:
```suggestion
```

Comment (fsansalvadore) [reply packages/common/hooks/index.ts:11]: duplicate

**Pull request merged**